doc/mf6io/mf6io_copy.bbl

```diff
@@ -0,0 +1,289 @@
+\begin{thebibliography}{41}
```

Comment:
do we need this file?

Reply:
Fixed with fd6d84c.  This file was inadvertently staged and committed during a previous commit.